src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -1249,7 +1248,7 @@ private void ValidateAttachment()
 
     private void ValidateClientVersionForItemLevelEncryptionSupport(Cipher cipher)
```

Comment:
:warning: We actually need to remove the whole method and calls to it.

Reply:
BAH thank you! updated